test/printf.c

```diff
@@ -0,0 +1,20 @@
+#include <stdio.h>
```

Comment:
Yes, but make sure you optimize (e.g. `-Os`) , as this doesn't happen without opts.